src/components/breadcrumb/index.js

```diff
@@ -1,6 +1,7 @@
 import bBreadcrumb from './breadcrumb';
 import bBreadcrumbItem from './breadcrumb-item';
 import bBreadcrumbLink from './breadcrumb-link';
+import registerComponent from '../../utils';
```

Comment:
Shoudn't it be `{ register component }` ?

Reply:
Yep, all fixed now ;-)